lib/go/edgecontext/edgecontext.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/gofrs/uuid"
```

Comment:
We might want to use `google/uuid`? https://github.com/google/uuid

Reply:
We have been using this uuid package across the existing bp.go code. See: https://github.com/reddit/edgecontext/blob/4f1c53c76fc417815b10ff90c1d27d25603fba8f/go.mod#L7